### PR TITLE
Extract Review: Fix typos + remove unused variable

### PR DESCRIPTION
**Pull request by BigRoy**
## Changelog Description

Cleanup code: remove unused variable `pixel_aspect` and fix typos

## Additional info

n/a

## Testing notes:

1. Extract review should work.

### CODE REVIEW

File: client/ayon_core/plugins/publish/extract_review.py [hunk 1]
```diff
@@ -1228,16 +1228,6 @@ def rescaling_filters(self, temp_data, output_def, new_repre):
         reformat_in_baking = bool("reformated" in new_repre["tags"])
```
Comment (MustafaJafar) [inline client/ayon_core/plugins/publish/extract_review.py:1228]: I was going to push a commit but I don't know what would happen if I changed that line.
```suggestion
        reformat_in_baking = bool("reformatted" in new_repre["tags"])
```

**Review by MustafaJafar — COMMENTED** [review]
Code-spell has reviewed your PR.

```
PS E:\Ynput\ayon-core> ./.poetry/bin/poetry run codespell client/ayon_core/plugins/publish/extract_review.py
client/ayon_core/plugins/publish/extract_review.py:622: ouput ==> output
client/ayon_core/plugins/publish/extract_review.py:958: witht ==> with
client/ayon_core/plugins/publish/extract_review.py:970: tempalte ==> template
client/ayon_core/plugins/publish/extract_review.py:1000: representaion ==> representation
client/ayon_core/plugins/publish/extract_review.py:1228: reformated ==> reformatted
client/ayon_core/plugins/publish/extract_review.py:1548: Containg ==> Containing
client/ayon_core/plugins/publish/extract_review.py:1862: beggining ==> beginning
7
```

File: client/ayon_core/plugins/publish/extract_review.py (hunk 1 above)
Comment (BigRoy) [reply client/ayon_core/plugins/publish/extract_review.py:1228]: Ignoring this in favor of this PR: https://github.com/ynput/ayon-core/pull/285

Comment (iLLiCiTiT) [reply client/ayon_core/plugins/publish/extract_review.py:1228]: Handled in separate PR https://github.com/ynput/ayon-core/pull/285

**Comment by BigRoy** [comment]
> Code-spell has reviewed your PR.
> 
> ```
> PS E:\Ynput\ayon-core> ./.poetry/bin/poetry run codespell client/ayon_core/plugins/publish/extract_review.py
> client/ayon_core/plugins/publish/extract_review.py:622: ouput ==> output
> client/ayon_core/plugins/publish/extract_review.py:958: witht ==> with
> client/ayon_core/plugins/publish/extract_review.py:970: tempalte ==> template
> client/ayon_core/plugins/publish/extract_review.py:1000: representaion ==> representation
> client/ayon_core/plugins/publish/extract_review.py:1228: reformated ==> reformatted
> client/ayon_core/plugins/publish/extract_review.py:1548: Containg ==> Containing
> client/ayon_core/plugins/publish/extract_review.py:1862: beggining ==> beginning
> 7
> ```

Thanks, fixed with https://github.com/ynput/ayon-core/pull/312/commits/413f30456f339193e73a18b6a3dad85d22022b48

Except `reformated` - skipped that in favor of this PR: https://github.com/ynput/ayon-core/pull/285

**Review by iLLiCiTiT — APPROVED** [review]

**Pull request merged**